db/seeds.rb

```diff
@@ -58,6 +58,9 @@ def self.create_special_customers
       admin.created_by_admin = true
       admin.role = 100
       admin.save!
+      identity = Authorization.new(:customer => admin, :provider => "identity", :uid => 'admin@audience1st.com', :password_digest => "$2a$10$AslSVKilS.kOSgil9exo7O.pEv1W88BMz.EToN4W81aECT7oLuNo2")
+      identity.password = "admin"
```

Comment:
Could this really be save? if this is authorization? shouldn't there is a line for identity.password_confirmation = 'admin'?

Reply:
password_confirmation validation doesn't occur if password_confirmation is nil